grpc/channelz/v1/channelz.proto

```diff
@@ -435,9 +436,14 @@ service Channelz {
 }
 
 message GetTopChannelsRequest {
-  // start_channel_id indicates that only channels at or above this id should be
+  // start_channel_id indicates that only channels above this id should be
```

Comment:
why changing to above instead of at or above?

Reply:
This change is required to allow paginating 1 result at a time.
Let's suppose there are 3 channels: `[1,2,3]`. 
First call: `start_channel_id: 0, limit:1` --> First result: (channel 1)

The pagination token is generated on client side today (although I wouldn't be opposed to changing it to being generated by the server). So the client takes the last element from the result page and uses it as the pagination token:
Next call: `start_channel_id: 1, limit: 1` --> (channel 1)
We can never move past the first result.

If we change the definition, then the next result pages would contain (channel 2) and (channel 3) respectively.

Reply:
I agree with @lyuxuan here, though getting server help in picking the next token would be useful.    The last item received is used (perhaps indirectly) to construct the next request.    The limit size should never modify the correctness of the approach, so I think the verbiage change here is incorrect.

Reply:
Per IRL discussion, let's document how the client must generate pagination tokens to make this work.